docs/loadflow.rst

```diff
@@ -1,8 +1,99 @@
 Load Flow
 =========
 
+.. testsetup:: *
+
+    import pypowsybl as pp
+
+You can use the module ``pypowsybl.loadflow`` in order to run load flows on networks. By default, load flows are based on the OpenLoadFlow implementation, fully described on `Powsybl website <https://www.powsybl.org/pages/documentation/simulation/powerflow/openlf.html>`_. OpenLoadFlow supports AC Newton-Raphson and linear DC calculation methods.
+
+Parameters
+**********
+
+The most important part before running a load flow is, after importing a network, knowing the parameters and change it if needed. Let's have a look at the default ones:
+
+.. doctest::
+
+    >>> network = pp.network.create_eurostag_tutorial_example1_network()
+    >>> import pypowsybl.loadflow
+    >>> pp.loadflow.Parameters()
+    LoadFlowParameters(voltage_init_mode=UNIFORM_VALUES, transformer_voltage_control_on=False, no_generator_reactive_limits=False, phase_shifter_regulation_on=False, twt_split_shunt_admittance=False, simul_shunt=False, read_slack_bus=False, write_slack_bus=False, distributed_slack=True, balance_type=PROPORTIONAL_TO_GENERATION_P_MAX, dc_use_transformer_ratio=True, countries_to_balance=[], connected_component_mode=<ConnectedComponentMode.MAIN: 0>)
+
+
+All parameters are fully described in `Powsybl loadfow parameter documentation <https://www.powsybl.org/pages/documentation/simulation/powerflow/>`_.
+
+- *voltage_init_mode* represents the starting point mode: use ``pp.loadflow.VoltageInitMode.`` and then UNIFORM_VALUES for a flat start, and DC_VALUES for a DC load flow based starting point.
```

Comment:
It's probably a good place for this for now, but I was thinking that it would maybe be more "efficient" in the future to write those descriptions in the docstring directly:
that way it will be available in IDEs, and also in the doc generated from the docstring (that we could print here, or point to).